packages/fxa-auth-server/bin/key_server.js

```diff
@@ -154,7 +154,7 @@ function run(config) {
             }
           }
 
-          init();
```

Comment:
Sorry, I'll rephrase. Git blame shows the last change to this line from 4 months ago during prettier. If this line must include `return`, how did we miss that for 4 months? What wasn't working correctly? Is there a test we can include that will prevent repeat breakage?

Reply:
Ok, I located where this came from, which I'm sure @vladikoff can appreciate 😉 :
https://github.com/mozilla/fxa/commit/98ad6719b643637c1a60add100e718393178bfbf#diff-1c5b95cf27fd1faec187d6374b32bdabL123

If the `return` is necessary, I'm a bit flummoxed how we didn't notice that for over a year.

Reply:
It's possible this wasn't an issue before because there wasn't anything pressuring the race condition. Now the tests initialize faster because they're doing less by not starting up an oauth-server. The error originated here https://github.com/mozilla/fxa/blob/master/packages/fxa-auth-server/test/lib/server.js#L63 because `server` was `null`. It is weird that I only saw this today since it should have shown up in earlier oauth->auth phase 2 changes. Maybe my machine likes the cooler weather.

Reply:
This is a great 'why' to include in the commit!